client/my-sites/hosting/main.js

```diff
@@ -78,7 +79,15 @@ const Hosting = ( { translate } ) => {
 								'Manage your databases with PHPMyAdmin and run a wide range of operations with MySQL.'
 							) }
 						</p>
-						<Button>{ translate( 'Access PHPMyAdmin' ) }</Button>
+						{ ( ! atomicPmaNonce || ! siteId ) && <Spinner /> }
+						{ atomicPmaNonce && siteId && (
+							<a
```

Comment:
Using a link here for now instead of Calypso button as by default when opening external link in new tab the Button component adds noopener/noreferer for security reasons. In this case given we are linking to a known external site the referer details could be useful to us - open to discussion on this - another option is to open this link in the same tab/window rather than a new one. Open to suggestions about the best way to handle this.